### ISSUE TITLE
Add filter for discord invites.

### ISSUE THREAD

**Issue by ammongit** [ivandardi/RustbotPython#1]
Filter messages posted by non-moderators or whitelisted individuals, by checking for the existence of substrings like `discord.gg`, `discord.me`, and `discordapp.com/invite`.

An example regex that could suffice for this:
```regex
(https?:\/\/)*discord(app)?\.(gg|me|com)\/\w+
```

**Comment by kangalioo** [comment]
I don't think this is a good idea, to be honest. There are many discord servers relevant to Rust that are brought up sometimes, for example The Coding Den or the Serenity Discord.
